core/common/src/test/java/alluxio/resource/DynamicResourcePoolTest.java

```diff
@@ -30,7 +30,8 @@
 public final class DynamicResourcePoolTest {
 
   /**
-   * Resource class used to be test DynamicResourcePool.
+   * Resource class used to test {@link DynamicResourcePool} and referenced by
```

Comment:
It's okay to just use ` Resource class used to test {@link DynamicResourcePool}`.

The remaining part was my explanation :)